docs/src/pages/components/data-grid/rows/rows.md

```diff
@@ -53,6 +53,15 @@ Alternatively, if you would like to delete a row, you would need to pass an extr
 apiRef.current.updateRows([{ id: 1, _action: 'delete' }]);
 ```
 
+### Throttling grid update [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
```

Comment:
AG-Grid uses the "high frequency" expression. I found it simpler to understand. BTW, this entire "Updating rows" section is not well structured I think. It starts as:

Updating rows - "...there're two ways": 
- 1st way: the `rows` prop
- Infinite loading?
- 2nd way: using the imperative API

We can make this new title one level up, because it's not only updating the rows, but also doing it in an async manner.

```suggestion
## High frequency [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
```

Reply:
I agree that the structure of this documentation page is not great, we can improve it in a follow up PR ?